core/src/main/resources/reference.conf

```diff
@@ -1,6 +1,6 @@
 rasterframes {
   nominal-tile-size = 256
-  prefer-gdal = true
+  prefer-gdal = false
```

Comment:
i like it. will simplify a lot of use cases particularly for folks new to the project.